standard/docs/locale/es/LC_MESSAGES/schema/merging.po

```diff
@@ -285,10 +285,10 @@ msgstr ""
 
 #: ../../standard/docs/en/schema/merging.md:127
 msgid ""
-"The [versioned release validation schema](../../versioned-release-"
+"The [versioned release validation schema](../../../../versioned-release-"
 "validation-schema.json) provides a model for representing this data."
 msgstr ""
-"El [esquema de validación de entrega versionada](../../versioned-release-"
+"El [esquema de validación de entrega versionada](../../../versioned-release-"
```

Comment:
Weird that for a working link, we need 4 `../` in the English, but 3 in the Spanish translation.